Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM mono:6.12.0
+
+# Install .NET SDK
+ENV DOTNET_VERSION=5.0
+
+RUN curl -sSL https://dot.net/v1/dotnet-install.sh | bash /dev/stdin -Channel $DOTNET_VERSION -InstallDir /usr/share/dotnet \
+    && ln -s /usr/share/dotnet/dotnet /usr/bin/dotnet
+    
+WORKDIR /SharpHoundCommon
+
+CMD [ "dotnet", "build" ]
+
+## Build Docker image (one time):
+# docker build -t sharphoundcommon-build . --no-cache
+
+## Build solution (every time):
+# docker run --rm -v "$(pwd):/SharpHoundCommon" sharphoundcommon-build
```

Comment:
this works for the volume mount, but you should also be able to use `.` for the current directory, making the quotes unnecessary: `-v .:/SharpHoundCommon`